src/main/java/pl/kzcwat/localincidentserver/region/RegionFactory.java

```diff
@@ -0,0 +1,34 @@
+package pl.kzcwat.localincidentserver.region;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import pl.kzcwat.localincidentserver.region.exception.RegionNotFoundException;
+import pl.kzcwat.localincidentserver.region.request.RegionRequest;
+
+@RequiredArgsConstructor
+@Service
+public class RegionFactory {
```

Comment:
```suggestion
public class RegionMapper {
```

Fabryka to bardziej przy wzorcu fabryki, tutaj może lepiej nazwać `RegionMapper`, mapuje dto<->entity?